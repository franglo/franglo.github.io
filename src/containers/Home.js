import React from 'react';
import { connect } from 'react-redux';
import Header from '../components/Header';

const mapStateToProps = ({ projects }, ownProps) => {
  return {
    projects: projects.list
  }
};
â€Š
const mapDispatchToProps = (dispatch, ownProps) => {
  return {
  }
};

const Home = props => (
  <div>
    <Header />
  </div>
);

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(Home);
