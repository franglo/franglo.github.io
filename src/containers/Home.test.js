import React from 'react';
import configureMockStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import Home from './Home';

const mockStore = configureMockStore();
const store = mockStore({
  projects: {
    list: [
      { name: 'Project #1' },
      { name: 'Project #2' },
      { name: 'Project #3' }
    ]
  }
});

test('', () => {
  const component = renderer.create(
    <Home store={store} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
