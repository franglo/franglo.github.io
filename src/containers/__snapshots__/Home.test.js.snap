// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div>
    <header
      className="sc-bdVaJa gkupbu"
    />
  </div>
</div>
`;
